Sources/TuistCore/Utils/Printer.swift

```diff
@@ -38,14 +38,14 @@ public class Printer: Printing {
 
     public func print(error: Error) {
         let writer = InteractiveWriter.stderr
-        writer.write("❌ Error: ", inColor: .red, bold: true)
+        writer.write("Error: ", inColor: .red, bold: true)
         writer.write(error.localizedDescription)
         writer.write("\n")
     }
 
     public func print(success: String) {
         let writer = InteractiveWriter.stdout
-        writer.write("✅ Success: ", inColor: .green, bold: true)
```

Comment:
What's the reason for removing these emoji's?

Reply:
I removed it because the color and the prefix is enough to denote the type of message being printed.

Reply:
There's a subtle difference when using `tuist` directly or via `tuistenv` - the colours are sadly lost in the latter case